dask/dataframe/reshape.py

```diff
@@ -3,7 +3,7 @@
 
 from .core import Series, DataFrame, map_partitions, apply_concat_apply
 from . import methods
-from .utils import is_categorical_dtype, is_scalar, has_known_categories
```

Comment:
It looks like there is another function in this module using `is_categorical_dtype`. Removing this import results in `NameError: name 'is_categorical_dtype' is not defined` being raised

Reply:
Yes - fixing this locally right now, should just be `is_categorical_dtype` -> `methods.is_categorical_dtype` I believe.